scripts/git/patches/boringssl/0001-disable-warnings.patch

```diff
@@ -1,25 +1,13 @@
-From 831806231dd360278dac2a37e3c3158420ad7bb3 Mon Sep 17 00:00:00 2001
```

Comment:
With the patch hand-edited, I had to remove the Git-specific stuff from it and make it a normal patch file.